software/main.py

```diff
@@ -87,31 +89,38 @@ def main_loop():
 
     try:
         while True:
-            print("------------------------- Current state: ", state, "-------------------------")
-            speed_T=0
+            #print("------------------------- Current state: ", state, "-------------------------")
+            speed_T = 0
+            grabber = 6000

```

Comment:
These numbers are hard to understand. Maybe if they are a range of -1 to 1 we can understand the logic better. 0 = no motion and thrower ramp is centered while -1 is a low/high (depends witch way you like it) position on the ramp and reverse on the thrower motor and 1 is the opposite